lib/TPP/DefaultPipeline.cpp

```diff
@@ -74,6 +80,18 @@ PrintStage parsePrintStage(StringRef stage) {
       .Default(PrintStage::None);
 }
 
+std::vector<int64_t> parseParallelTaskGrid(std::string parallelTaskGrid) {
```

Comment:
Don't do this, use `llvm::cl::list<int64_t>` in the cmd-line option.

https://github.com/plaidml/tpp-mlir/blob/00dbee7c39d7576d2b8aa86624fd98aef4bd4d8d/lib/TPP/GPU/GpuPipeline.cpp#L49

Reply:
Fixed